packages/elastic-safer-lodash-set/lodash/_baseSet.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is forked from the lodash project (https://lodash.com/),
+ * and may include modifications made by Elasticsearch B.V.
+ * Elasticsearch B.V. licenses this file to you under the MIT License.
+ * See `packages/elastic-safer-lodash-set/LICENSE` more information.
```

Comment:
Minor typo in this and similar license headers: "**for** more information"

Reply:
Fixed 👍